src/engine/source/indexerconnector/src/monitoring.hpp

```diff
@@ -62,9 +62,14 @@ enum Type : std::size_t
 
 /**
  * @brief Monitoring class.
+ * This class is used to monitor the health of the servers.
```

Comment:
Nitpick: Doxygen supports notes. There are appended below the brief
```suggestion
 *
 * @note This class is used to monitor the health of the servers.
```

Reply:
Done!